packages/auth-construct/README.md

```diff
@@ -0,0 +1,133 @@
+# AmplifyAuth Construct
+
+This package vends an L3 CDK Construct that enables faster, easier and secure app authentication and authorization powered by Amazon Cognito. Amplify Auth lets you quickly set up secure authentication flows with a fully-managed user directory. Control what users have access to in your mobile and web apps with Amplify Auth's built-in authorization capabilities.
+
+The primary way to use this construct is to invoke it with a configuration object. You can declare the individual settings for your authentication by passing them as properties to the AmplifyAuth construct.
+
+Note: only a single instance of the `AmplifyAuth` construct can be invoked within a CDK synthesis at this point in time.
+
+## Examples
+
+### Simple email login with default settings
+
+In this example, you will create a simple stack with email login enabled (by default). Deploying this will create a UserPool, UserPoolClient, IdentityPool, and Authenticated/Unauthenticated IAM Roles.
```

Comment:
Should that be `a UserPool, a UserPoolClient, an IdentityPool, and an Authenticated/Unauthenticated IAM Role." or "a UserPool, a UserPoolClient, an IdentityPool, and Authenticated/Unauthenticated IAM Roles."?

Reply:
looks valid from a few grammar check sites